rdza_proc_macro/src/lib.rs

```diff
@@ -5,9 +5,9 @@ fn replace_ident(ident: Ident) -> Option<TokenTree> {
 
     let new_str = match ident_str.as_str() {
         "Źle" => "Err",
-        "Dobrze" => "Ok",
+        "Dobrze" | "Prawilnie" => "Ok",
```

Comment:
Myślę, że oprócz `Prawilnie` powinniśmy dodatkowo wspierać `Wporzo` oraz `Elegancko`:

```suggestion
        "Dobrze" | "Prawilnie" | "Wporzo" | "Elegancko" => "Ok",
```